### PR TITLE
Check size of workspace before uploading

### PR DESCRIPTION
**Pull request by MahdiZareie**

### CODE REVIEW

File: fandogh_cli/fandogh_client/__init__.py [hunk 1]
```diff
@@ -86,6 +91,9 @@ def get_image_build(image_name, version, token):
 
 
 def create_version(image_name, version, workspace_path):
+    workspace_size = os.path.getsize(workspace_path) / 1048576  # which is 1024 * 1024
```
Comment (s-soroosh) [inline fandogh_cli/fandogh_client/__init__.py]: this is not true. why don't you consider the .dockerignore file?

Comment (s-soroosh) [reply fandogh_cli/fandogh_client/__init__.py]: check the size of created zip file

Comment (s-soroosh) [reply fandogh_cli/fandogh_client/__init__.py]: from issue
`Ideally we should give some warning to the user if the workspace size is more than usual. e.g. > 20MB
and give him some hint how to use .dockerignore feature.`

so we don't want to stop uploading just some warnings a hints

**Comment by MahdiZareie** [comment]
@psycho-ir updated

**Review by s-soroosh — APPROVED** [review]

**Pull request merged**